assets/dev/js/editor/controls/popover-toggle.js

```diff
@@ -21,53 +21,27 @@ export default class ControlPopoverStarterView extends ControlChooseView {
 		this.$el.next( '.elementor-controls-popover' ).toggle();
 	}
 
-	behaviors() {
-		const behaviors = ControlChooseView.prototype.behaviors.apply( this, arguments );
-
-		// We don't want to override existing inherited behaviors, such as dynamic tags
-		if ( ! this.options.model.attributes.global ) {
-			return behaviors;
-		}
-
-		behaviors.globalControlSelect = {
-			behaviorClass: GlobalControlSelect,
-			popoverTitle: elementor.translate( 'global_text_styles_title' ),
-			manageButtonText: elementor.translate( 'manage_global_text_styles' ),
-			tooltipText: elementor.translate( 'global_typography_info' ),
-			newGlobalConfirmTitle: elementor.translate( 'create_global_color' ),
-		};
-
-		return behaviors;
-	}
-
-	createGlobalPreviewMarkup( textStyle ) {
-		// This method is called without a color parameter when the user clicks the "Add" button
+	createGlobalItemMarkup( textStyle ) {
+		// This method is called without a parameter when the user clicks the "Add" button
 		if ( ! textStyle ) {
 			textStyle = this.getTypographyObject();
 		}
 
-		const $textStylePreview = jQuery( '<div>', { class: 'elementor-global-preview elementor-global-text-style', 'data-elementor-global-name': textStyle.name } );
+		const $textStylePreview = jQuery( '<div>', { class: 'e-global-preview e-global-text-style', 'data-elementor-global': textStyle.value } ),
+			{ fontFamily, fontSize, fontWeight, transform, fontStyle, textDecoration, lineHeight, letterSpacing } = textStyle;
```

Comment:
Is there a better way to move the parameters immediately to the css() ?

Reply:
Couldn't find a better way